services/web/server/src/simcore_service_webserver/folders/_folders_api.py

```diff
@@ -282,4 +282,11 @@ async def delete_folder(
         workspace_id=folder_db.workspace_id,
     )
 
-    await folders_db.delete(app, folder_id=folder_id, product_name=product_name)
+    # 1. Delete folder
+    # 1.1 Delete all child projects that I am an owner?
+    # NOTE: not yet implemented needs to be discussed
+
+    # 1.2 Delete all child folders
+    await folders_db.delete_with_all_children(
+        app, folder_id=folder_id, product_name=product_name
+    )
```

Comment:
my 2 cents here:
- as a user if I delete a folder I expect everything inside it to be deleted, but,
- I want to have a pop-up that asks me if I am really sure?
- or if there is another option to move my folders somewhere.. but that is a bit weird (at least from my point of view)

Reply:
I agree with this